api/.env.example

```diff
@@ -27,7 +27,11 @@ PERMISSION_MAP='{
 S3_ENDPOINT="your.s3.endpoint"
 S3_PORT=9000
 S3_USE_SSL=true
-S3_ACCESS_KEY="accesskey"
-S3_SECRET_KEY="secretkey"
-S3_IMG_BUCKET="bucketname"
+S3_ACCESS_KEY="minio"
```

Comment:
I hope this is just for testing purpose otherwise it might be a faillure security access

Reply:
Just placeholder info. But trying to make it work by default for testing and dev purposes